### PR TITLE
Task 3.2

### PR DESCRIPTION
**Pull request by 01gaM**
@toshiks 

Best validation accuracy: 0.90, best hyperparams: Hyperparams(learning_rate=0.1, anneal_epochs=50, reg=1e-05)

Final test accuracy - 0.9005454824830977

### CODE REVIEW

**Comment by 01gaM** [comment]
@toshiks 
Task 6

LSTMTagger:
Loss = 43.08183, Accuracy = 94.41%

Bidirectional LSTM:
Loss = 21.53829, Accuracy = 96.56%

Качество модели с предобученной матрицей на тестовой выборке:
Loss = 14.22690, Accuracy = 96.73%
